ext/libgav1.cmd

```diff
@@ -9,7 +9,7 @@
 : #     "C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\VC\Auxiliary\Build\vcvars64.bat"
 
 # When updating the libgav1 version, make the same change to libgav1_android.sh.
-git clone -b v0.18.0 --depth 1 https://chromium.googlesource.com/codecs/libgav1
+git clone -b v0.19.0 --depth 1 https://chromium.googlesource.com/codecs/libgav1
```

Comment:
Please also update tests/docker/build.sh. Thanks.

Reply:
And CHANGELOG.md.

Reply:
Done in https://github.com/AOMediaCodec/libavif/pull/1893 !